pkg/intermodule/account.ts

```diff
@@ -150,3 +147,8 @@ export const dummyAccountModuleFacade = new AccountModuleFacade(
       .feed(Ether.compose(inMemoryAccountRepository)).value,
   ),
 );
+
+export const accountModuleEther = () =>
```

Comment:
```suggestion
export const accountModuleEther =
```
`Ether` 自体が遅延してオブジェクトを構築しているので, 関数に包む必要はないと思います.
タイムラインの方のものも同様です.